content/metadata.yaml

```diff
@@ -37,4 +37,4 @@ authors:
       - Department of Pharmacology, University of Colorado School of Medicine
       - Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine
     funders:
-      - Grant XXXXXXXX
+      - TODO
```

Comment:
- The Gordon and Betty Moore Foundation (GBMF4552)
      - The National Human Genome Research Institute (R01 HG10067)